src/commands/config-set.json

```diff
@@ -0,0 +1,33 @@
+{
+    "SET": {
+        "summary": "Set a configuration parameter to the given value",
+        "complexity": "PATCH__TBD__47__",
```

Comment:
this is gonna change soon in #9748 (not sure which one is gonna get merged first)

Reply:
probably 9748 - let's try to remember to update this json when the other PR is merged

Reply:
actually this PR changes the current command table so once one of the PRs is merged the other will get conflicts